src/ToggleButton/ToggleButton.tsx

```diff
@@ -0,0 +1,134 @@
+/* ======================================================================== *
+ * Copyright 2024 HCL America Inc.                                          *
+ * Licensed under the Apache License, Version 2.0 (the "License");          *
+ * you may not use this file except in compliance with the License.         *
+ * You may obtain a copy of the License at                                  *
+ *                                                                          *
+ * http://www.apache.org/licenses/LICENSE-2.0                               *
+ *                                                                          *
+ * Unless required by applicable law or agreed to in writing, software      *
+ * distributed under the License is distributed on an "AS IS" BASIS,        *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. *
+ * See the License for the specific language governing permissions and      *
+ * limitations under the License.                                           *
+ * ======================================================================== */
+
+import React from 'react';
+import MuiToggleButton, { ToggleButtonProps as MuiToggleButtonProps } from '@mui/material/ToggleButton';
+import { Components, Theme } from '@mui/material/styles';
+
+export enum ToggleButtonVariants {
+  WITHOUT_PADDING = 'without padding',
+  WITH_PADDING = 'with padding',
+}
+
+export enum ToggleButtonSizes {
+  SMALL = 'small',
+  MEDIUM = 'medium',
+}
+
+/**
+ * @typedef ToggleButtonProps
+ * @type {object}
+ * @property {ToggleButtonSizes} size - The size of the component
+ * @property {ToggleButtonVariants} variant - Adds padding around icon svg
+ */
+export interface ToggleButtonProps extends MuiToggleButtonProps {
+  size?: ToggleButtonSizes,
+  variant?: ToggleButtonVariants,
+  color?: 'standard',
+}
+
+const ToggleButton = React.forwardRef(({ ...props }: ToggleButtonProps, forwardRef) => {
+  return <MuiToggleButton {...props} ref={forwardRef as ((instance: HTMLButtonElement | null) => void)} aria-disabled={props.disabled} />;
+}) as React.FC<ToggleButtonProps>;
+
+ToggleButton.defaultProps = {
+  size: ToggleButtonSizes.SMALL,
+  variant: ToggleButtonVariants.WITH_PADDING,
+  disabled: false,
+  selected: false,
+  color: 'standard',
```

Comment:
standard seems a bit weird, should it be `default` or something? like I see in other components

Reply:
the options for color are these: https://mui.com/material-ui/api/toggle-button/#toggle-button-prop-color. it does not have 'default' like the other components